app/views/home/index.html.erb

```diff
@@ -9,8 +9,8 @@
     <header class="clearfix">
       <h1>Flashcards</h1>
       <ul class="navigation-links">
-        <li><a href="#">All Cards</a></li>
-        <li><a href="#">Add Card</a></li>
+        <li><a href="/cards/index">All Cards</a></li>
```

Comment:
@kennyfrc please, use the helper paths, run `rake routes` and you get all routes in your application + `_path` for working